docs/ref/contrib/gis/functions.txt

```diff
@@ -576,6 +576,20 @@ SpatiaLite
 Accepts a single geographic field or expression and returns a geometry with
 reversed coordinates.
 
+``Rotate``
+===========
+
+.. class:: Rotate(expression, angle, x=0.0, y=0.0, origin=None **extra)
+
+*Availability*: `PostGIS <https://postgis.net/docs/ST_Rotate.html>`__
+
+Accepts a single geographic field or expression and returns a geometry with
+coordinates rotated by the specified ``angle`` around the origin.
+Optionally, the rotation can be performed around a specified point, defined by
+either the ``x`` and ``y`` parameters or the ``origin`` parameter.
```

Comment:
I had to research to understand this method, so perhaps we could add a bit more of information. I will make a push with a proposed expansion.

Question: did you consider having the `x` and `y` params to be a single tuple param for clarity? cc/@claudep

Reply:
EDIT: on a second look, `Scale` (and `Translate`) use the `x`, `y` and `z` params so it feels like `x` and `y` are consistent here.